airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4418,6 +4497,11 @@ void loop(void) {
 			data += result_GPS;
 			sum_send_time += sendLuftdaten(result_GPS, GPS_API_PIN, F("GPS"), "GPS_");
 		}
+		else if(cfg::mhz19_read)
+		{
+			data += result_MHZ19;
+			sum_send_time += sendLuftdaten(result_MHZ19, GPS_API_PIN, F("MHZ19"), "MHZ19_");
```

Comment:
Copy and paste error here - the gps api pin needs to be changed to something mhz19 specific

Reply:
I was using the same api pin number as the gps because it actually uses the same hardware pin.
I didn't understand the concept of the "api pin".
What's is it used for and what value shall I use?

Reply:
I found [some info about the API pin](https://github.com/opendata-stuttgart/sensors-software/tree/master/airrohr-firmware#luftdateninfo-api-pins), but still not clear what it is and which number to chose

Reply:
I can't really tell you what the exact purpose is either..my guess is that this is some way to tag the information and match it in the server side backend. However, even id that's untrue, I would just define a new #define to the same value then instead of repeating the name here..

Reply:
> tag the information and match it in the server side backend.

yes, it is only used in sendData():
```
	request_head += F("X-PIN: ");
	request_head += pin;

```

I will define a new number 16 (one after DNMS)

Reply:
Please use 17 as pin.
The esp8266 chipID together wuth this pin is used to identify the component for which the data is sent. We try to use the same pin for sensors with the same purpose. So all PM sensors use pin 1, the temperature/pressure sensors pin 3, temperature/humidity pin 7, temperature/humidity/pressure pin 11, GPS pin 9 (more in ext_def.h)
Some users might modify the firmware to use 2 sensors of the same type (i.e. temperature/humidity inside or outside the casing).  For such a device we could add a sensor with pin 7 and one with pin 8.